classes/render/assets.php

```diff
@@ -1081,6 +1080,10 @@ protected static function set_locale_code($locale)
 			$locale = strtolower(substr($locale, 0, 2));
 		} elseif (file_exists(CFCORE_PATH . 'assets/js/i18n/' . strtolower(substr($locale, 3)) . '.js')) {
 			$locale = strtolower(substr($locale, 3));
+		} else {
+			//No file is matching the locale in validation folder, 
+			//Fallback to english instead of not enqueueing for Parsley to be set a locale and not return an error
```

Comment:
Me: It smells bad that there is nothing in the else, but nothing is wrong.

Also me: It smelled bad for a reason.

Reply:
Agree